kernelci/cli/node.py

```diff
@@ -52,7 +53,7 @@ def find(attributes, config, api,   # pylint: disable=too-many-arguments
     api = get_api(config, api)
     attributes = split_attributes(attributes)
     nodes = api.get_nodes(attributes, offset, limit)
-    data = json.dumps(nodes, indent=indent)
+    data = json.dumps(nodes, indent=indent or None)
```

Comment:
`indent` will be set to `None` by default if not provided.
Also, `indent=indent or None` would be evaluated to `None` if user has provided `0` as indent.
```suggestion
    data = json.dumps(nodes, indent=indent)
```

Reply:
No, I made this change on purpose as otherwise it prints the output formatted with newlines and no identation space when `indent=0`.  So `None` is needed in this case.

Reply:
It's also explained in the git commit and the `echo_json()` docstring.